src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/HistoricDataVisualizer.java

```diff
@@ -62,8 +62,8 @@ public void start(Stage primaryStage) throws IOException {
         // original: writer = new EthereumWriter(Path.of(OUTPUTDIRECTORY), filename);
         writer = new EthereumWriter(Path.of(OUTPUTDIRECTORY), filename);
         BlockAggregator dbs = initDataBlockSummerizer(filename);
-        TestCenter testCenter = new TestCenter(client, new File(OUTPUTDIRECTORY + "extractedData6.json"));
-        testCenter.generateJsonFile(0, 10000);
+        TestCenter testCenter = new TestCenter(client, new File(OUTPUTDIRECTORY + "transactionsFrom100kBlocks2.json"));
+        testCenter.generateJsonFile(0, 1000);
```

Comment:
As said before, we should refrain from committing changes to this file, if changes are only done for local testing.

Reply:
ok then i will change the outputfile back to a standard file everytime before committing.

Reply:
You can simply not commit this change. GIT does not require you to commit all changes in your working directory.

Reply:
FYI, conversations should be resolved by the reviewer.